test/cypress/e2e/integration/app-frontend/mobile.js

```diff
@@ -35,8 +35,8 @@ describe('Mobile', () => {
     cy.contains(mui.button, texts.next).click();
     cy.get(appFrontend.group.sendersName).should('be.visible').type('automation');
     cy.get(appFrontend.navMenu).should('have.attr', 'hidden');
-    cy.get('nav > button').should('be.visible').and('have.attr', 'aria-expanded', 'false').click();
-    cy.get('nav > button').should('have.attr', 'aria-expanded', 'true');
+    cy.get('#form-content-nav2 > div > nav > button').should('be.visible').and('have.attr', 'aria-expanded', 'false').click();
```

Comment:
Could the selector possibly be a bit more helpful in terms of explaining what we are selecting. Maybe we should make it a habit to wrap the selector strings in variables, to make these tests more readable?

Reply:
Good point, extracted to variable.